proto/stream_plan.proto

```diff
@@ -11,47 +11,56 @@ import "source.proto";
 
 option optimize_for = SPEED;
 
-message StopMutation {
-  repeated uint32 actors = 1;
+message AddMutation {
+  message Dispatchers {
+    repeated Dispatcher dispatchers = 1;
+  }
+  // New dispatchers for each actor.
+  map<uint32, Dispatchers> actor_dispatchers = 1;
+  // We may embed a source change split mutation here.
+  // TODO: we may allow multiple mutations in a single barrier.
+  map<uint32, source.ConnectorSplits> actor_splits = 2;
```

Comment:
I do not get why the mutation needs `ConnectorSplits`

Reply:
We need to provide the initialized source splits through this mutation

Reply:
By the way, is `actor_splits` only used for creating source? So we may distinguish creating MV on MV and creating source and use `SourceSplit` mutation for the latter one.

Reply:
No, `actor_splits` is not used when creating a source, it is only used in the following two places

1. the initialization barrier at the end of `create materialized view` will push the initial kafka partition down to the split reader in the source executor via `actor_splits`

2. source's config change, which is used to push down the new partition